llvm/lib/CAS/MappedFileRegionBumpPtr.cpp

```diff
@@ -52,12 +52,14 @@
 //===----------------------------------------------------------------------===//
 
 #include "llvm/CAS/MappedFileRegionBumpPtr.h"
-#include "llvm/ADT/ScopeExit.h"
+
```

Comment:
don't need empty line around this include?

Reply:
Hmm, I see a mix of styles for including lib-directory headers.  Some have empty lines around it, some only have empty line above it, some have no empty lines.  Not clear to me what it should be

Reply:
No spaces seems more common, going with that.